velox/type/Filter.cpp

```diff
@@ -219,12 +219,8 @@ xsimd::batch_bool<int64_t> BigintValuesUsingHashTable::testValues(
     return Filter::testValues(x);
   }
   auto allEmpty = xsimd::broadcast<int64_t>(kEmptyMarker);
-#if defined(__aarch64__)
   xsimd::batch<int64_t> indices(
-      *(reinterpret_cast<uint64x2_t*>(&x)) * M & sizeMask_);
-#else
-  xsimd::batch<int64_t> indices(xsimd::batch<uint64_t>(x) * M & sizeMask_);
-#endif
+      simd::reinterpretBatch<uint64_t>(x) * M & sizeMask_);
```

Comment:
This quite didn't work. The compilation of `Filter.cpp` failed. Can you take a look at it @Yuhta? Thanks

Reply:
What is the error message you get here?

Reply:
Here is the error message:
```c
Filter.cpp:223:58: error: no matching function for call to ‘xsimd::batch<long int>::batch(xsimd::batch<long unsigned int, xsimd::neon64>)’
  223 |       simd::reinterpretBatch<uint64_t>(x) * M & sizeMask_);
      |                                                          ^
```

Reply:
> What is the error message you get here? Is it still there if you fix the `vreinterpretq_u64_s64`?

Yes.

Reply:
Should be fixed now, can you check again?